### PR TITLE
sync with english version for MDN landing page

### PR DESCRIPTION
**Pull request by yin1999**
### Description

sync with english version for MDN landing page

### Related issues and pull requests

The landing page, a part of #11063

### CODE REVIEW

File: files/zh-cn/mdn/index.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
 slug: MDN
 ---
 
 {{MDNSidebar}}
 
-**MDN Web 文档** 是免费使用我们编辑的开放 web 平台文档的网站。我们的使命是为*开发者*提供*信息*，帮助他们轻松地在 *web 平台*上构建项目。
+**MDN Web 文档**是免费使用的，且允许我们记录开放 web 平台的资源。我们的使命是为*开发者*提供*信息*，帮助他们*轻松地*在 *web 平台*上构建项目。
 
-这是 MDN 项目的着陆页。在这里，您可以找到有关网站如何工作、如何编写文档、遵循哪些准则和惯例以及您如何提供帮助的指南。
+这是 MDN Web 文档项目的着陆页。在这里，你可以找到有关网站如何工作、如何编写文档、遵循哪些准则和惯例，以及你如何提供帮助的指南。
```
Comment (awxiaoxian2020) [inline files/zh-cn/mdn/index.md:10]:
```suggestion
这是关于 MDN Web 文档项目如何落地的页面。在这里，你可以找到有关网站如何工作、如何编写文档、遵循哪些准则和惯例，以及你如何提供帮助的指南。
```

File: files/zh-cn/mdn/index.md [hunk 2]
```diff
@@ -1,16 +1,25 @@
...
 
-同时，我们邀请任何人来帮忙！如果您有兴趣改进这个重要的 web 开发者资源，欢迎您添加和编辑内容。您不需要是一个程序员或了解很多技术，因为这里有许多不同的从简单的（校对和更正打字错误）到复杂的（编写 API 文档）任务需要执行。
+同时，我们邀请所有人来提供帮助！MDN Web 文档是一个开源的项目，并接受来自社区的贡献。这里有许多你可以提供帮助的，从简单（校对和纠正拼写错误）到复杂（撰写 API 文档）的任务。
```
Comment (awxiaoxian2020) [inline files/zh-cn/mdn/index.md:12]:
```suggestion
同时，我们邀请所有人来提供帮助！MDN Web 文档是一个开源的项目，并接受来自社区的贡献。这里有许多从简单（校对和纠正拼写错误）到复杂（撰写 API 文档）的任务，你都可以为此作出贡献。
```

File: files/zh-cn/mdn/index.md [hunk 3]
```diff
@@ -1,16 +1,25 @@
...
+如果要了解如何提供帮助，请访问我们的[入门](/zh-CN/docs/MDN/Contribute)页面。如果你想与我们交谈并提出问题，请加入 [Matrix](https://wiki.mozilla.org/Matrix) 上的 [MDN Web 文档聊天室](https://chat.mozilla.org/#/room/#mdn:mozilla.org)进行讨论。
 
-{{LandingPageListSubPages()}}
+- [社区准则](/zh-CN/docs/MDN/Community)
+
+  - : 这里的指南可以帮助你开始为 MDN Web 文档作贡献。它还涵盖了诸如如何帮助我们处理任务解决议题、开始一个讨论或建议内容等主题。如果你需要帮助或联系我们，你也可以在此处找到相关信息。
+
+- [撰写指南](/zh-CN/docs/MDN/Writing_guidelines)
+
+  - : 这里的指南提供了有关如何撰写 MDN Web 文档的所有信息。它们涵盖了我们撰写（及不撰写）的文档类型的相关政策。你还可以在这里找到我们的文档写作规范、处理各类内容任务的指南，以及有关我们不同页面结构的信息。
```
Comment (awxiaoxian2020) [inline files/zh-cn/mdn/index.md:22]:
```suggestion
  - : 这里的指南提供了有关如何撰写 MDN Web 文档的所有信息。这里有我们收录内容的相关政策。你还可以在这里找到我们的文档写作规范、处理各类内容任务的指南，还有关于我们不同页面结构的相关信息。
```

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: files/zh-cn/mdn/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/mdn/index.md:10]: landing page 是指对信息汇总的页面

Comment (awxiaoxian2020) [reply files/zh-cn/mdn/index.md:10]: > landing page 是指对信息汇总的页面

edited

Comment (yin1999) [reply files/zh-cn/mdn/index.md:10]: 着陆页这个翻译是比较常见的，不作修改 https://support.google.com/google-ads/answer/14086?hl=zh-Hans

File: files/zh-cn/mdn/index.md (hunk 2 above)
Comment (yin1999) [reply files/zh-cn/mdn/index.md:12]:
```suggestion
同时，我们邀请所有人来提供帮助！MDN Web 文档是一个开源的项目，并接受来自社区的贡献。这里有许多从简单（校对和纠正拼写错误）到复杂（撰写 API 文档）的任务，你可以通过这些任务为我们提供帮助。
```

**Pull request merged**